hadoop-ozone/dist/src/main/compose/ozone/docker-config

```diff
@@ -14,6 +14,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+CORE-SITE.XML_fs.defaultFS=ofs://om
+CORE-SITE.XML_fs.trash.interval=1
```

Comment:
Note: Those two lines need to be removed later as they are causing those robot test failures.
Leaving them here for now just for testing.